aspire-orchestrator/Aspire.Extensions/UvAppHostingExtensions.cs

```diff
@@ -43,8 +43,8 @@ private static IResourceBuilder<UvAppResource> AddUvApp(this IDistributedApplica
         // var projectExecutable = instrumentationExecutable ?? pythonExecutable;
 
         string[] allArgs = args is { Length: > 0 }
-            ? ["run", scriptPath, .. args]
-            : ["run", scriptPath];
+            ? ["run", "--frozen", scriptPath, .. args]
+            : ["run", "--frozen", scriptPath];
 
```

Comment:
@tommasodotNET FYI the `--frozen` param cannot be after `scriptPath` so I hardcoded it here

Reply:
We can leave it hard coded or, if it makes sense to have it chosen programmatically, we could pass a flag to this method and hard code the `--frozen` param only if the flag is true.